docs/cudf/source/Working-with-missing-data.ipynb

```diff
@@ -0,0 +1,3032 @@
+{
```

Comment:
<p>I think we might not be following pandas in this </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; pd.Series([np.nan], dtype='float64').sum(skipna=False)
nan

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>We have the same behavior in this with respect to pandas, no?:</p><p> </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; cudf.Series([np.nan], nan_as_null=False).sum(skipna=False)
nan
</pre><p> </p><p>Updated notebook with this example too.</p>

Reply:
I missed seeing `sum(skipna=False)` and thought `nan_as_null=False`. The behaviors are inline.